server/models/User.py

```diff
@@ -51,6 +57,11 @@ def update(self, data):
         for key, item in data.items():
             if key == 'password':
                 self.password = self.__generate_hash(item)
+
+            elif key == 'cuisines':
+                list_of_ids = list(map(lambda id: id['id'], item))
+                self.cuisines = Cuisine.get_cuisines_by_ids(list_of_ids)
+
```

Comment:
Might be less confusing to change the variable in the lambda function to "cuisine" or just "c" since item is a list of cuisines.

Reply:
Thanks! good catch.

Reply:
Just changed variable name to "cuisine"